source/releasenotes/compat.rst

```diff
@@ -56,6 +56,8 @@ and VMware with vSphere.
 
 -  VMware versions 6.5, 6.7 and 7.0
 
+   .. note:: The following VMware minor versions are supported: 6.5, 6.7, 6.7.1, 6.7.2, 6.7.3, 7.0, 7.0.1.0, 7.0.2.0, 7.0.3.0. For the minor versions not in this list, all virtual machines have guest OS identifier "otherGuest64" (x86-64 architecture) or "otherGuest" (other architectures).
```

Comment:
@weizhouapache do you want to include 6.5, 6.7, 7.0 in this minor versions list?

Reply:
@sureshanaparti 
yes
7.0 = ESXi 7.0 GA on https://kb.vmware.com/s/article/2143832